sdk/identity/azure-identity/src/main/java/com/azure/identity/ManagedIdentityCredential.java

```diff
@@ -48,14 +48,27 @@
         Configuration configuration = identityClientOptions.getConfiguration() == null
             ? Configuration.getGlobalConfiguration().clone() : identityClientOptions.getConfiguration();
 
+
+        /*
+         * Choose credential based on available environment variables in this order:
+         *
+         * Azure Arc: IDENTITY_ENDPOINT, IMDS_ENDPOINT
+         * Service Fabric: IDENTITY_ENDPOINT, IDENTITY_HEADER, IDENTITY_SERVER_THUMBPRINT
+         * App Service 2019-08-01: IDENTITY_ENDPOINT, IDENTITY_HEADER (MSI_ENDPOINT and MSI_SECRET will also be set.)
```

Comment:
For my understanding, `will also be set`, does this mean this will set those variables or that those variables will need to be set to be able to use `App Service`

Reply:
Good question. They are the legacy variables for the older endpoint and they're always set in those cases for backward compat reasons.